distribution/helm/vector-agent/templates/daemonset.yaml

```diff
@@ -13,10 +13,9 @@ spec:
     type: {{ .Values.updateStrategy }}
   template:
     metadata:
-    {{- with .Values.podAnnotations }}
       annotations:
-        {{- toYaml . | nindent 8 }}
-    {{- end }}
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        {{- with .Values.podAnnotations }}{{- toYaml . | nindent 8 }}{{ end }}
```

Comment:
Let's use multi-line notation here, like with `labels` below

Reply:
Sure, just looked a little funny to me

Reply:
I wonder if we even need this?